dist/units/units.scss

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Seeing as all the content under `/dist` is generated by the build scripts - do we need to commit it to the repo? It may be cleaner and easier to maintain if we only include the source?